mixed_requests/test_mixed.py

```diff
@@ -60,6 +60,11 @@ def create_servers(self):
         port = tempesta.upstream_port_start_from()
         self.servers = [control.Nginx(listen_port=port)]
 
+    # wrk problem
```

Comment:
Not clear comment. What wrk problem is mentioned here. It took me some time to remember it, but for others it'll be definitely not clear.